extension-config.json

```
@@ -0,0 +1,13 @@
+{
+  "version": "1.0.0",
+  "id": "@shopgate-project/open-external-link",
+  "components": [
+    {
+      "id": "externalLink",
```

```suggestion
      "id": "ExternalLink",
```
afaik other widget use upper case first letter

---

Updated